### PR TITLE
Remove unclear hosting model language

### LINKED ISSUE #20073: Confusing words on Blazor hosting models
># Blazor app hosting models
>Blazor apps **can be hosted in IIS just like ASP.NET Web Forms apps**. Blazor apps **can also be hosted in one of the following ways**:
> * Client-side in the browser on WebAssembly.
> * Server-side in an ASP.NET Core app.

It is confusing to describe in this way, because how many hosting models are there? Two or three? If there are only two hosting models (as the following of this article indicates), then the first two sentences should be revised to remove ambiguity.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4b45d194-8874-5bd1-31eb-767145461904
* Version Independent ID: 9817ff2d-afe1-8a5f-1e1a-7c7eadba5cf0
* Content: [Blazor app hosting models](https://docs.microsoft.com/en-us/dotnet/architecture/blazor-for-web-forms-developers/hosting-models#feedback)
* Content Source: [docs/architecture/blazor-for-web-forms-developers/hosting-models.md](https://github.com/dotnet/docs/blob/master/docs/architecture/blazor-for-web-forms-developers/hosting-models.md)
* Product: **dotnet-architecture**
* Technology: **blazor**
* GitHub Login: @danroth27
* Microsoft Alias: **daroth**

### PR DESCRIPTION
**Pull request by danroth27**
Fixes https://github.com/dotnet/docs/issues/20073

### CODE REVIEW

File: docs/architecture/blazor-for-web-forms-developers/hosting-models.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ ms.date: 11/20/2020
 ---
 # Blazor app hosting models
 
-Blazor apps can be hosted in IIS just like ASP.NET Web Forms apps. Blazor apps can also be hosted in one of the following ways:
+Blazor apps can also be hosted in one of the following ways:
```
Comment (BartoszKlonowski) [inline docs/architecture/blazor-for-web-forms-developers/hosting-models.md:11]: Should "*also*" stay here, if the first way has been removed?

Comment (danroth27) [inline docs/architecture/blazor-for-web-forms-developers/hosting-models.md:11]:
```suggestion
Blazor apps can be hosted in one of the following ways:
```

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**